assets/js/race.js

```diff
@@ -28,6 +28,7 @@ function removeIndividual(item) {
 }
 
 /*
+<<<<<<< HEAD
```

Comment:
This is a conflict sign, please remove it (every time after you resolve the conflicts).